index.d.ts

```diff
@@ -46,6 +46,11 @@ export interface FastifyDynamicSwaggerOptions extends FastifySwaggerOptions {
   mode?: 'dynamic';
   swagger?: Partial<SwaggerSchema.Spec>;
   hiddenTag?: string;
+  /**
+   * Strips matching base path from routes in documentation
+   * @default true
+   */
+  stripBasePath?: boolean;
```

Comment:
We also have a test for the types, can you add this option there as well? Thanks

Reply:
I pushed a small addition to the tests, but I'm not sure if it's what you were asking for.